### PR TITLE
Adds default-election-id capabilities

### PR DESCRIPTION
**Pull request by tank157**
[Pivotal Card](https://www.pivotaltracker.com/story/show/160763099)

### CODE REVIEW

File: src/cljs/gttp2/events.cljs [hunk 1]
```diff
@@ -88,6 +88,16 @@
   (when links
     (config/update-links links)))
 
+(defn default-election-id
+  "Pulls an optional default-election-id from the environment."
+  []
+  (when-let [election-id (optional-env-var :default-election-id)]
+    {:election-id election-id}))
+
+(defn maybe-assoc
```
Comment (ddellacosta) [inline src/cljs/gttp2/events.cljs]: Don't we have this somewhere else? Maybe we should pull it out into its own generic `utils` kinda namespace?

Comment (tank157) [reply src/cljs/gttp2/events.cljs]: We did have it another place, in the same file though. I just moved it higher.

Comment (ddellacosta) [reply src/cljs/gttp2/events.cljs]: Oh whoops, nevermind then!

Comment (tank157) [reply src/cljs/gttp2/events.cljs]: I mean, it's not a bad suggestion to move it to utils, let me see if there's a better spot for it.

**Pull request merged**